forms-flow-web/src/components/PrivateRoute.jsx

```diff
@@ -221,7 +221,7 @@ const PrivateRoute = React.memo((props) => {
         <Route
           {...rest}
           render={(props) =>
-            createDesigns || viewDesigns || createSubmissions  ? (
+            createDesigns || viewDesigns || createSubmissions || viewSubmissions  ? (
```

Comment:
This will allow user with viewSubmissions access to form list page ? @arun-s-aot @abilpraju-aot .

Reply:
tested in local and that form module enable only when the view_submission permission enable in navbar

Reply:
@fahad-aot Please review